test/extended/clusterup.sh

```diff
@@ -299,16 +313,30 @@ echo "Running cluster up tests using tag $ORIGIN_COMMIT"
 docker pull openshift/origin-docker-registry:latest
 docker tag openshift/origin-docker-registry:latest openshift/origin-docker-registry:${ORIGIN_COMMIT}
 
+# Tag the web console image with the same tag as the other origin images
+docker pull openshift/origin-web-console:latest
+docker tag openshift/origin-web-console:latest openshift/origin-web-console:${ORIGIN_COMMIT}
+
 # Ensure that KUBECONFIG is not set
 unset KUBECONFIG
 for test in "${tests[@]}"; do
-  for extra_arg in "${extra_args[@]}"; do
-	cleanup_func=$("os::test::extended::clusterup::cleanup_func" "${test}")
-	# trap "${cleanup_func}; os::test::extended::clusterup::junit_cleanup" EXIT
-    os::test::extended::clusterup::run_test "${test}" "${extra_arg}"
-    # trap - EXIT
-	${cleanup_func}
-  done
+    # Special case the console tests, which only run in the current release.
+    # N-1 tests for the console should be enabled next release.
+    if [ "$test" == "console" ]; then
+        cleanup_func=$("os::test::extended::clusterup::cleanup_func" "${test}")
+        # trap "${cleanup_func}; os::test::extended::clusterup::junit_cleanup" EXIT
+        os::test::extended::clusterup::run_test "${test}" "--loglevel=2"
```

Comment:
pass the --version here instead of hardcoding it in your test.  that way when someone removes this special case things will "just work" the way they're supposed to.